packages/fxa-content-server/app/scripts/templates/post_verify/newsletters/add_newsletters.mustache

```diff
@@ -7,7 +7,13 @@
                 {{/isSignedIn}}
             {{/showSuccessMessage}}
 
-            {{#t}}Sign up for more{{/t}}<span class="description">{{#t}}Practical knowledge is coming to your inbox.{{/t}}</span>
+            {{#isTrailheadCopy}}
+                {{#t}}Sign up for more{{/t}}<span class="description">{{#t}}Practical knowledge is coming to your inbox.{{/t}}</span>
+            {{/isTrailheadCopy}}
+
+            {{^isTrailheadCopy}}
+                {{#t}}Sign up for more{{/t}}<span class="description">{{#t}}Get practical knowledge in your inbox about:{{/t}}</span>
+            {{/isTrailheadCopy}}
```

Comment:
Nit, not required for merge but could be slightly DRYer:

```
{{#t}}Sign up for more{{/t}}
<span class="description">
  {{#isTrailheadCopy}}
    {{#t}}Practical knowledge is coming to your inbox.{{/t}}
  {{/isTrailheadCopy}}
  {{^isTrailheadCopy}}
    {{#t}}Get practical knowledge in your inbox about:{{/t}}
  {{/isTrailheadCopy}}
</span>
```
Also fine if you prefer the `<span class="description">copy</span>` in the conditional instead of splitting the open span.

This is outside of your PR, but I don't think `This Firefox is connected` should be in the `h1`, especially when it's displayed _with_ the text here. It's probably more like a `div` with `role="alert"` 🤔